src/brackets.js

```diff
@@ -425,7 +425,31 @@ define(function (require, exports, module) {
                 node = node.parentElement;
             }
         }, true);
-        
+
+        // on Windows, cancel every other scroll event (#10214)
+        // TODO: remove this hack when we upgrade CEF to a build with this bug fixed:
+        // https://bitbucket.org/chromiumembedded/cef/issue/1481
+        var winCancelWheelEvent = true;
```

Comment:
I wonder if it would make it feel more responsive to start with this 'false'?  That way the first scroll in the pair is actually rendered, the second is ignored.  It didn't feel at all laggy in my testing, but it might be worth trying out...